src/mito/src/table.rs

```diff
@@ -529,9 +516,44 @@ impl<R: Region> MitoTable<R> {
         Ok(table_info)
     }
 
+    /// Remove regions
+    /// Notes: Please release regions in StorageEngine.
+    pub async fn remove_regions(&self, region_numbers: &[RegionNumber]) -> TableResult<()> {
+        self.regions.rcu(|regions| {
+            let mut regions = HashMap::clone(regions);
+            for region in region_numbers {
+                regions.remove(region);
```

Comment:
If a region is removed from `regions`, will it be dropped by Rust? There might be no other references to it.

Reply:
> other

The `MitoEngineInner` still keeps it.